### PR TITLE
Add ElasticSearch + Filebeat 5.0 instructions on CentOS

### PR DESCRIPTION
**Pull request by fernandosanchezmunoz**
## What type of changes does your doc introduce?
- [X] Bug fix

## What is the urgency level of this change?
- [ ] Blocker
- [X] High
- [ ] Medium

## I have completed these items:
- [X] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [X] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

CC: @emanic , @sascala , or @joel-hamill  for reviewing pull request changes.
This is alligned with the method suggested by @malnick for 1.9 and onwards on PR: https://github.com/dcos/dcos-docs/pull/726
This is valid for 1.8 while Jeff's will be valid  1.9 an above. So likely keeping both make sense.

Also I haven't tested much for style and formatting so suggestions/edits are welcome.

### CODE REVIEW

File: 1.8/administration/logging/elk.md [hunk 1]
```diff
@@ -1,147 +1,200 @@
 ---
-post_title: Log Management with ELK
-nav_title: ELK
+post_title: Log Management in DC/OS with ELK
 menu_order: 1
+enterprise: 'yes'
 ---
-You can pipe system and application logs from a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server.
+
+You can pipe system and application logs from the nodes in a DC/OS cluster to your existing ElasticSearch, Logstash, and Kibana (ELK) server. This article recommends to store all unfiltered logs directly on ElasticSearch, and perform filtering/specialized querying on ElasticSearch directly. Consequently, the suggested pipeline sends the output from filebeat in the nodes directly to a centralized ElasticSearch instance, without using Logstash. If you're interested in using Logstash for log processing or parsing, consult the [Filebeat][2] and [Logstash][8] documentation.
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Suggest: `The filebeat output from each node is sent directly to a centralized ElasticSearch instance, without using Logstash.`

File: 1.8/administration/logging/elk.md [hunk 2]
```diff
@@ -1,147 +1,200 @@
...
+
+NOTE: This article does NOT cover how to set up secure TLS communication between the Filebeat instances and ElasticSearch. For details on how to achieved this, please check the [Filebeat][2] and [ElasticSearch][5] documentation.
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Suggest: 

- `**Important:** This document does not describe how to set up secure TLS communication...`
- `For information on how to achieve this, see the...`

File: 1.8/administration/logging/elk.md [hunk 3]
```diff
@@ -1,147 +1,200 @@
...
 
 # <a name="all"></a>Step 1: All Nodes
 
 For all nodes in your DC/OS cluster:
 
-1.  Install Elastic's [logstash-forwarder][2]. The project is written in Go and is compiled for most platforms.
-2.  Download the HTTPS certificate for your ELK stash. In our examples, we assume the certificate is named `elk.crt`.
+1.  Install Elastic's [filebeat][2]. Installers are available for most major platforms.
+
+```bash
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Indent this code block with the number, to render properly.

File: 1.8/administration/logging/elk.md [hunk 4]
```diff
@@ -1,147 +1,200 @@
...
+curl -L -O https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-5.0.0-x86_64.rpm
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Global: Add `$` before commands. This is our established convention throughout the docs.

File: 1.8/administration/logging/elk.md [hunk 5]
```diff
@@ -1,147 +1,200 @@
...
+sudo rpm -vi filebeat-5.0.0-x86_64.rpm
+```
+
+2. Create a filebeat configuration file to monitor all Mesos log files. Also create the `/var/log/dcos` directory, and add another input entry for the file `/var/log/dcos/dcos.log` that we will funnel the DC/OS logs into in a later step. Please make sure to substitute the variables $ELK_HOSTNAME and $ELK_PORT below for the values of the host/port where your ElasticSearch is listening on.
+
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Indent the code block to match the numbered item.

File: 1.8/administration/logging/elk.md [hunk 6]
```diff
@@ -1,147 +1,200 @@
...
+sudo systemctl daemon-reload
+sudo systemctl start dcos-journalctl-filebeat.service
+sudo chkconfig dcos-journalctl-filebeat.service on
+sudo systemctl start filebeat
+sudo chkconfig filebeat on
+```
+
+# <a name="all"></a>Step 3: ELK Node Notes
+
+The ELK stack will receive, store, search and display information about the logs parsed by the Filebeat instances configured above for all nodes in the cluster. In this example, we are streaming directly from Filebeat into ElasticSearch, so Logstash is not used in this architecture. If you're interested in filtering, parsing and grok'ing the logs with an intermediate Logstash stage, please check the Logstash [documentation][8].
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Suggest: `The ELK stack will receive, store, search and display parsed log information for all nodes in the cluster that are configured for Filebeat.`

Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Suggest: `In this document, we are streaming directly from Filebeat into ElasticSearch, so Logstash is not used in this architecture. `

File: 1.8/administration/logging/elk.md [hunk 7]
```diff
@@ -1,147 +1,200 @@
...
+
+There are many possible architectural options to install and run the ELK stack, including running it inside DC/OS itself as a service. This article does not cover the full scope of installing ElasticSearch and Kibana For further details on other installation options, please consult the ELK Stack [documentation][4]
+
+1. Important parameters to configure in the ElasticSearch node.
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: I think this sentence can deleted.

File: 1.8/administration/logging/elk.md [hunk 8]
```diff
@@ -1,147 +1,200 @@
...
+
+Some parameters will need to be changed from their default values for ElasticSearch to be ready to receive information. Edit the ElasticSearch configuration file (typically `/etc/elasticsearch/elasticsearch.yml`) and edit:
+
+
+```
+network.host = [IP address from the interface in your ElasticSearch node connecting to the Filebeat instances]
+```
+
+Other parameters in the file are beyond the scope of this document. For details, please check the ElasticSearch [documentation][5]
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: Add period after `documentation][5]`.

File: 1.8/administration/logging/elk.md [hunk 9]
```diff
@@ -1,147 +1,200 @@
...
+
 
 ### Known Issue
 
-*   The agent node logstash configuration expects tasks to write logs to `stdout` and `stderr`. Some DC/OS services, including Cassandra and Kafka, do not write logs to `stdout` and `stderr`. If you want to log these services, you must customize your agent node logstash configuration.
+*   The agent node filebeat configuration expects tasks to write logs to `stdout` and `stderr`. Some DC/OS services, including Cassandra and Kafka, do not write logs to `stdout` and `stderr`. If you want to log these services, you must customize your agent node filebeat configuration.
```
Comment (joel-hamill) [inline 1.8/administration/logging/elk.md]: `The agent node Filebeat configuration...`

File: 1.8/administration/logging/elk.md (hunk 1 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK

File: 1.8/administration/logging/elk.md (hunk 2 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 4 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 5 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 6 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md [hunk 10]
```diff
@@ -1,147 +1,200 @@
...
+sudo chkconfig dcos-journalctl-filebeat.service on
+sudo systemctl start filebeat
+sudo chkconfig filebeat on
+```
+
+# <a name="all"></a>Step 3: ELK Node Notes
+
+The ELK stack will receive, store, search and display information about the logs parsed by the Filebeat instances configured above for all nodes in the cluster. In this example, we are streaming directly from Filebeat into ElasticSearch, so Logstash is not used in this architecture. If you're interested in filtering, parsing and grok'ing the logs with an intermediate Logstash stage, please check the Logstash [documentation][8].
+
+There are many possible architectural options to install and run the ELK stack, including running it inside DC/OS itself as a service. This article does not cover the full scope of installing ElasticSearch and Kibana For further details on other installation options, please consult the ELK Stack [documentation][4]
```
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 7 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 8 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

File: 1.8/administration/logging/elk.md (hunk 9 above)
Comment (fernandosanchezmunoz) [inline 1.8/administration/logging/elk.md]: OK modified accordingly.

**Comment by fernandosanchezmunoz** [comment]
@joel-hamill suggested changes are added -- let me know if anything else needs to be done here.

**Comment by joel-hamill** [comment]
Looks good to me. Thanks!

**Comment by joel-hamill** [comment]
@fernandosanchezmunoz actually, one last item, can you add this to the 1.9 branch too?

**Comment by fernandosanchezmunoz** [comment]
@joel-hamill done! my understanding is that at some point, @malnick and his team will have updates to this, but at least this is working so it's better than what we had.

**Pull request merged**